doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Change Data Capture (Postgres)"
+description: "How to create real-time materialized views from PostgreSQL changelog data with Materialize."
+weight: 10
+menu:
+  main:
+    parent: guides
+---
+
+Change Data Capture (CDC) allows you to track and propagate changes in a database to downstream consumers based on its changelog. In this guide, we'll cover how to use Materialize to create and efficiently maintain real-time materialized views on top of Postgres CDC data.
+
+There are two ways to connect Materialize to a Postgres database for CDC:
+
+* Using the direct [Postgres source](#direct-postgres-source)
+
+* Using [Kafka + Debezium](#kafka--debezium)
+
+## Direct Postgres source
+
+If Kafka is not part of your stack, you can use the [Postgres source](/sql/create-source/postgres/#postgresql-source-details) to connect directly to Materialize. This source uses Postgres’ native replication protocol to continuously ingest changes resulting from `INSERT`, `UPDATE` and `DELETE` operations in the upstream database.
+
+### Setup
+
+**Minimum requirements:** Materialize v0.8.0+, Postgres 10+
```

Comment:
0.8.0 has postgres behind the experimental flag and with an incompatible syntax from what was stabilized. We should go for `v0.8.2`